### ISSUE TITLE
exclude nodes , weird behavior

### ISSUE THREAD

**Issue by laurentdebricon** [rkfg/regolancer#16]
hey, 
super great script  !!! but, i think i have found a weird behavior with **exclude_nodes** , with this configuration, it says "no route", but with the second version using **exclude_channels** behavior is correct.
```
"fee_limit_ppm"= 500
"amount"= 10000
"min_amount"= 1000
"probe_steps"= 5
"pfrom"= 20
"pto"= 80
"stat"= "stats.csv"
 "exclude_nodes"= [
     "033d8656219478701227199cbd6f670335c8d408a92ae88b962c49d4dc0e83e025", # bfx-lnd0
     "02cc125c2770dd293f56b50841c8d847276efd00634d48e947c9b83125315d8f", # bfx-lnd1
     "021c97a90a411ff2b10dc2a8e32de2f29d2fa49d41bfbb52bd416e460db0747d0d", # loop
     "024bfaf0cabe7f874fd33ebf7c6f4e5385971fc504ef3f492432e9e3ec77e1b5cf" # deezy
]
"to" = [
  831055968312229888
]
"from" = [
    
]
lost_profit = true
allow_unbalance_to = true
allow_unbalance_from = true
```
not working, 

but this is working : 
```
"fee_limit_ppm"= 500
"amount"= 10000
"min_amount"= 1000
"probe_steps"= 5
"pfrom"= 20 
"pto"= 80
"stat"= "stats.csv"
"exclude_channels"= [
        796000239066611713, #bfx0
        832584289538146304, # bfx0
        831646406136823809, #bfx1
        832612876845056000,  #deezy
        833668408015519745 #deezy
]
"to" = [
  831055968312229888
]
"from" = [
    
]
lost_profit = true
allow_unbalance_to = true
allow_unbalance_from = true
```

**Comment by rkfg** [comment]
Looks like you're missing 2 characters in `bfx-lnd1` pubkey, it's shorter than the rest. I think this situation isn't handled correctly, will take a look later.

**Comment by rkfg** [comment]
Should be fixed in 465b61829cd05bcf907459c2fb6754d539f1cc8e, now you'll get an error on startup regarding that invalid node id.
